### PR TITLE
Fix Perfect Model Ice Nucleation Calibrations

### PR DESCRIPTION
**Pull request by amylu00**
- Changed standard dev to be more than 1 for priors.
- More reasonable outputs/initial conditions.
- Frozen fraction vs time plots for ABDINM and ABIFM to check outputs.

to be merged only after AIDA HOM calibrations are merged!

### CODE REVIEW

File: test/ice_nucleation_calibration.jl
Comment (trontrytel) [inline test/ice_nucleation_calibration.jl:35]: Is there a reason why one value is checked with relative and the other with absolute tolerance?

File: papers/ice_nucleation_2024/calibration.jl
Comment (trontrytel) [inline papers/ice_nucleation_2024/calibration.jl:267]: Is this really needed? 150?

**Review by trontrytel — APPROVED** [review]

File: parcel/ParcelModel.jl
Comment (amylu00) [reply parcel/ParcelModel.jl:20]: When droplets or crystals are set to lognormal distribution. Lognormal distribution wasn't tested anywhere so I didn't catch that it was missing earlier. Not specifically related to this PR

File: papers/ice_nucleation_2024/calibration.jl
Comment (amylu00) [reply papers/ice_nucleation_2024/calibration.jl:267]: Think I needed it before when the calibrations weren't working right and things didn't converge early enough. I can change it to 75 but if I set it to anything lower, one of the experiments doesn't seem to really converge (IN05_19)

File: test/ice_nucleation_calibration.jl
Comment (amylu00) [reply test/ice_nucleation_calibration.jl:35]: The c coefficient can be pretty small so using relative tolerance up to 0.70 won't pass the test even though the value is not horribly calibrated (i.e. true c coefficient for ABDINM is ~2 but calibrated c will be ~4 which doesn't seem too bad to me), so I switched to atol. The m coefficient is ~20 to ~255 so it's easier to use rtol and have the tests pass. 

To force the tests to pass using rtol I could:

1. Make the noise of the pseudo data really small and there will be a higher probability that the test for the c coefficient will pass for say rtol = 0.3, but there's still a chance that the calibration will decide to have a relatively large or small guess for a specific ensemble (plus at that point, I'm just making the pseudo-data noiseless which seems to defeat the purpose of the perfect model tests in the first place).
2. Make the standard deviation for the priors very small which would also defeat the purpose of calibration since I'm basically giving it the true value to start with.

At some point I was tired of restarting the CI checks until the tests pass so I changed the c coefficient test to atol instead of rtol. If you have a cooler fix please let me know 🙃

**Comment by amylu00** [comment]
> Left some small comments, feel free to squash and merge afterwards.
> 
> I'm probably not best equipped to review calibrations setup. It might be good to ask @odunbar when he is back to take a look. But parcel model wise everything looks good!

Thanks, Anna! I should've rebased this branch onto the other AIDA hom branch but there's a lot of changes here that should be attached to PR #385 instead of this one. I've tagged Ollie in that one just now, but I think I still need your approval before GitHub lets me merge that one

**Pull request merged**
